vnext/Microsoft.ReactNative.ProjectReunion/Microsoft.ReactNative.ProjectReunion.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
I would probably recommend targeting 18362 (or 17763 if you want lowest) to be consistent with what other reunion projects are doing.  The main reason is that if you target 19041, due to assembly version bindings, consumers will not be able to consume this projection using a lower Windows TFM version even if it in theory supported.  If you do have dependencies on APIs that are only in 19041, then you should keep what you have.

Reply:
@manodasanW when I used 18362, I got 4 kB empty DLLs out of it :-/  https://github.com/microsoft/CsWinRT/issues/892

Reply:
ok I see why. there was a warning that it couldn't use microsoft.reactnative.winmd for 18362, because it is built for 19041, even though the min version is set to 17763. Added a comment to the issue I filed on cswinrt about this. In the meantime I'll try to fix the SDK version for the purpose of reunion in an upcoming PR. 
It would be good if cswinrt could produce an error in this case, as I never want an empty projection :)

Reply:
FYI it looks like 18362 is the minimum we can use because of IXP. If I set 17763, I get this:
```
The nuget command failed with exit code(1) and error(NU1202: Package Microsoft.ProjectReunion.InteractiveExperiences 0.8.0-preview is not compatible with net5.0-windows10.0.17763 (.NETCoreApp,Version=v5.0). Package Microsoft.ProjectReunion.InteractiveExperiences 0.8.0-preview supports:
  - net5.0-windows10.0.18362 (.NETCoreApp,Version=v5.0)
  - uap10.0 (UAP,Version=v10.0)
Errors in D:\a\_work\1\s\vnext\Microsoft.ReactNative.ProjectReunion\Microsoft.ReactNative.ProjectReunion.csproj
    NU1202: Package Microsoft.ProjectReunion.InteractiveExperiences 0.8.0-preview is not compatible with net5.0-windows10.0.17763 (.NETCoreApp,Version=v5.0). Package Microsoft.ProjectReunion.InteractiveExperiences 0.8.0-preview supports:
      - net5.0-windows10.0.18362 (.NETCoreApp,Version=v5.0)
      - uap10.0 (UAP,Version=v10.0))
````